### ISSUE TITLE
cubestore persistency with remoteDir doesn't work on gcp

### ISSUE THREAD

**Issue by roshan8** [gadsme/charts#24]
Hello, 

`standard` storageClass on gcp with readwrite access type won't be able to provision volume as the `standard` storage class doesn't support mounting volume on multiple pods in GCP.

**Comment by roshan8** [comment]
Thanks for your response. 

I did encode the gcp service account json with base64 and passed it as `CUBESTORE_GCP_CREDENTIALS`. However, cube unable to decode it and throws the following error:
```sh
2023-05-30 21:57:13,670 ERROR [cubestore::remotefs::gcs] <pid:1> Could not decode 'CUBESTORE_GCP_CREDENTIALS': Internal: Invalid byte 10, offset 3080.
```

I also tried to encode service account JSON file 2 times before creating the secret. as all the secrets need to base64 encoded and even cube expects `CUBESTORE_GCP_CREDENTIALS` to be in encoded format.
